### PR TITLE
CourseScripts model

### PR DESCRIPTION
**Pull request by Bjvanminnen**
We want to be able to define what set of scripts are in a course, and in what order. This model serves to do just that.

### CODE REVIEW

File: dashboard/db/migrate/20170505234020_create_course_scripts.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CreateCourseScripts < ActiveRecord::Migration[5.0]
+  def change
+    create_table :course_scripts do |course_script|
+      course_script.belongs_to :course, {null: false}
+      course_script.belongs_to :script, {null: false}
+      course_script.integer :position
```
Comment (Bjvanminnen) [inline dashboard/db/migrate/20170505234020_create_course_scripts.rb]: This should actually probably also be {null: false}. Will fix

File: dashboard/app/models/course.rb [hunk 2]
```diff
@@ -16,4 +16,5 @@
 class Course < ApplicationRecord
   # Some Courses will have an associated Plc::Course, most will not
   has_one :plc_course, class_name: 'Plc::Course'
+  has_many :course_scripts
```
Comment (ashercodeorg) [inline dashboard/app/models/course.rb:19]: nit: Model changes (also FactoryGirl changes) should be in a separate PR.

File: dashboard/db/migrate/20170508234536_create_course_scripts.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class CreateCourseScripts < ActiveRecord::Migration[5.0]
+  def change
+    create_table :course_scripts do |course_script|
+      course_script.belongs_to :course, {null: false}
+      course_script.belongs_to :script, {null: false}
+      course_script.integer :position, {null: false}
```
Comment (ashercodeorg) [inline dashboard/db/migrate/20170508234536_create_course_scripts.rb:6]: This is the position of the script within the course?

Comment (Bjvanminnen) [reply dashboard/db/migrate/20170508234536_create_course_scripts.rb:6]: Yes

Comment (aoby) [reply dashboard/db/migrate/20170508234536_create_course_scripts.rb:6]: You could add a comment explaining what the column is for.

Something like:
```ruby
course_script.integer :position, {null: false}, comment: 'position of the script with the course'
```
